src/market/market_utils.cairo

```diff
@@ -1030,18 +1039,50 @@ fn get_open_interest_with_pnl(
     0
 }
 
-
-/// Get the virtual inventory for swaps
-/// # Arguments
-/// * `data_store` - The data store to use.
-/// * `market` - The market address.
-/// # Returns
-/// has virtual inventory, virtual long token inventory, virtual short token inventory
+// @dev get the virtual inventory for swaps
+// @param data_store DataStore
+// @param market the market to check
+// @return returns (has virtual inventory, virtual long token inventory, virtual short token inventory)
```

Comment:
Why did you use this format for documenting

Reply:
was copied from solidity, fixed now